tests/sentinel/run.tcl

```diff
@@ -14,7 +14,7 @@ proc main {} {
         set ::env(LEAKED_FDS_FILE) $::leaked_fds_file
     }
     spawn_instance sentinel $::sentinel_base_port $::instances_count [list "sentinel deny-scripts-reconfig no"] "../tests/includes/sentinel.conf"
-    spawn_instance redis $::redis_base_port $::instances_count
+    spawn_instance redis $::redis_base_port $::instances_count [list "enable-protected-configs yes" "enable-debug-command yes"]
```

Comment:
considering this list already contains two configs, and may contain more some day, let's use similar syntax to the one in `cluster/run.tcl` (i.e. multiline with curly braces).

Reply:
done. i also had the same thought before (Just want to unify with the previous line)